database/sql/src/main/scala/cromwell/database/sql/joins/MetadataJobQueryValue.scala

```diff
@@ -0,0 +1,18 @@
+package cromwell.database.sql.joins
+
+sealed trait MetadataJobQueryValue
+
+/**
+  * Applies a filter to retrieve only metadata keys matching this job
+  */
```

Comment:
Names like "CallQuery" or "WorkflowQuery" or "CallOrWorkflowQuery" might be more clear?